nautobot/extras/tests/test_datasources.py

```diff
@@ -341,8 +234,7 @@ def create_empty_repo(path, url):
                     (job_result.result, list(job_result.job_log_entries.values_list("message", "log_object"))),
                 )
                 self.repo.refresh_from_db()
-                self.assertEqual(self.repo.current_head, self.COMMIT_HEXSHA, job_result.result)
-                MockGitRepo.assert_called_with(os.path.join(tempdir, self.repo.slug), "http://localhost/git.git")
+                # self.assertEqual(self.repo.current_head, self.COMMIT_HEXSHA, job_result.result)
```

Comment:
No longer needed?

Reply:
yeah, since we're not mocking the repo creation now, we don't have a well-known sha to compare the head against. I suppose we could grab the sha from the other repo and compare it - think that'd be worthwhile?

Reply:
The job log entry lookup below seems to be sufficient for testing this. We can probably just remove this assertion